spec/forms/select_mobile_number_form_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe SelectMobileNumberForm do
+  let(:claim) { double("Claim", teacher_id_user_info: {"phone_number" => "123-456-7890"}) }
```

Comment:
Could we use `instance_double` to ensure this test breaks if `teacher_id_user_info` is ever removed? I would use pure doubles when mocking external interfaces. As an alternative, you could use FactoryBot's `build_stubbed`

Reply:
Good idea. `instance_double` is used now instead 👍